debian/qualcoder.postinst

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
```

Comment:
unfamiliar with the postinst, but ok

Reply:
it updates the desktop database so that qualcoder will appear in the applications menu of the desktop.

Reply:
OK I understand better now. (Poor me,  beginner/intermediate developer) :)